UI/GameScreen.cpp

```diff
@@ -119,9 +120,7 @@ void GameScreen::CreateViews() {
 	rightColumnItems->Add(btnDeleteSaveData_)->OnClick.Handle(this, &GameScreen::OnDeleteSaveData);
 	btnDeleteSaveData_->SetVisibility(V_GONE);
 
-	if (info && !info->pending) {
-		otherChoices_.clear();
-	}
+	otherChoices_.clear();
```

Comment:
Ah, maybe this was it.  If otherChoices were added before somehow, and the cache was cleared (can an app switch do that?), then we might have stale but deleted pointers there.

-[Unknown]